.github/workflows/test.yml

```diff
@@ -7,99 +7,51 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-
-      - name: Install node
-        uses: actions/setup-node@v1
-        with:
-          node-version: "14.x"
-          registry-url: "https://registry.npmjs.org"
-
-      - name: Install Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.x"
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install twine wheel jupyter-packaging jupyterlab
-
-      - name: Build
-        run: |
-          python setup.py generate_source
-          python setup.py sdist bdist_wheel
-
-      - name: Build
-        run: |
-          cd js
-          npm pack
-
-      - name: Upload builds
-        uses: actions/upload-artifact@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
-          name: ipyvuetify-dist-${{ github.run_number }}
-          path: |
-            ./dist
-            ./js/*.tgz
```

Comment:
Please resture this, so we can download the build wheels, and test it locally.

Reply:
done as well